apps/stats-web/src/app/blocks/[height]/errors.tsx

```diff
@@ -1,11 +1,14 @@
 "use client"; // Error components must be Client Components
 
 import { useEffect } from "react";
+import { LoggerService } from "@akashnetwork/logging";
+
+const blockErrorLogger = new LoggerService({ context: "block" });
```

Comment:
this is generally used for dev. So we need to be able to look it up faster. prob smth like this would make sense more

```suggestion
const blockErrorLogger = new LoggerService({ context: "apps/stats-web/src/app/blocks/[height]/errors.tsx" });
```

wdut?